test/act-mapping/aria-allowed-attr.json

```
@@ -0,0 +1,5 @@
+{
+  "id": "5c01ea",
+  "title": "Aria attributes are used validly on elements depending on their semantic role",
```

The title should be the name of the ACT rule.

```suggestion
  "title": "ARIA state or property is permitted",
```